.gitignore

```diff
@@ -348,6 +348,7 @@ npm-debug.log*
 yarn-error.log*
 package-lock.json
 yarn.lock
+.env
```

Comment:
Eventually I'm not using `.env` file, keeping it just in case.

Reply:
why not use https://www.npmjs.com/package/dotenv?

Reply:
Environment variables are taken from the Angular project that ng deploy runs on. In our case the variables are relevant to ng deploy itself